resources/integration.rb

```diff
@@ -0,0 +1,57 @@
+# Install/remove a Datadog integration
+# This resource basically wraps the datadog-agent integration command to be able
+# to install and remove integrations from a Chef recipe.
+# The datadog_resource must be used on a system where the datadog-agent has
+# already been setup.
+
+default_action :install
+
+property :name, String, name_attribute: true
+
+property :version, String, required: true
+
+action :install do
+  unless node['datadog']['agent6']
+    log "The datadog_integration resource is only available with Agent v6." do
+      level :error
+    end
+    return
+  end
+
+  log "Getting integration #{new_resource.name}" do
+    level :debug
+  end
+
+  execute 'integration install' do
```

Comment:
might require running with `elevated: true` on windows? right? I'm not sure but be sure to test out.

Reply:
Good catch. We can't really use `elevated: true` because it would mean to ask for user credentials to execute this. I've added a note for Chef Windows users that the running chef-client must have read access to the `datadog.yaml`file.